### ISSUE TITLE
Graphql version 15.3 support 

### ISSUE THREAD

**Issue by ccsv** [apollographql/apollo-server#4624]
Version: 

- Apollo: 2.31.0
- graphql 15.3.0

Run `npm audit fix` after installing:

```
npm WARN @apollographql/graphql-language-service-interface@2.0.2 requires a peer of graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @apollographql/graphql-language-service-parser@2.0.2 requires a peer of graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @apollographql/graphql-language-service-types@2.0.2 requires a peer of graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @apollographql/graphql-language-service-utils@2.0.2 requires a peer of graphql@^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-link-persisted-queries@0.2.2 requires a peer of graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-subscriptions@1.1.0 requires a peer of graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-upload@8.1.0 requires a peer of graphql@0.13.1 - 14 but none is installed. You must install peer dependencies yourself.
```

Note the inconsistency in format for the graphql versioning in graphql-upload v. 8.1.0 

Running `npm list graphql`:

```
+-- UNMET PEER DEPENDENCY graphql@15.3.0
`-- vue-cli-plugin-apollo@0.22.2
  +-- apollo@2.31.0
  | +-- apollo-language-server@1.24.0
  | | `-- UNMET PEER DEPENDENCY graphql@15.3.0  deduped
  | `-- UNMET PEER DEPENDENCY graphql@15.3.0  deduped
  +-- apollo-server-express@2.18.1
  | `-- apollo-server-core@2.18.1
  |   `-- @types/graphql-upload@8.0.4
  |     `-- UNMET PEER DEPENDENCY graphql@15.3.0  deduped
  `-- UNMET PEER DEPENDENCY graphql@15.3.0  deduped

npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
npm ERR! peer dep missing: graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by graphql-subscriptions@1.1.0
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-parser@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
npm ERR! peer dep missing: graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by graphql-subscriptions@1.1.0
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-parser@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
npm ERR! peer dep missing: graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by graphql-subscriptions@1.1.0
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-parser@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
npm ERR! peer dep missing: graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by graphql-subscriptions@1.1.0
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-parser@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
npm ERR! peer dep missing: graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by graphql-subscriptions@1.1.0
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-interface@2.0.2
npm ERR! peer dep missing: graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by @apollographql/graphql-language-service-parser@2.0.2
npm ERR! peer dep missing: graphql@^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0, required by apollo-link-persisted-queries@0.2.2
```

**Comment by glasser** [comment]
Thanks! I'm going to handle this at #4865.

**Comment by glasser** [comment]
Apollo Server v2.21.0 will fix this.  I published alpha packages with version v2.21.0-alpha.0.  I'd love some folks to try the alpha and let me know if it works for you! Both folks who actually use the upload feature and those who don't! It's currently in the release-2.21.0 branch, being tracked at https://github.com/apollographql/apollo-server/pull/4922
